src/main/scala/org/clulab/wm/eidos/groundings/grounders/SRLCompositionalGrounder.scala

```diff
@@ -163,8 +173,111 @@ class SRLCompositionalGrounder(name: String, domainOntology: DomainOntology, w2v
 
     }
 
-    Seq(newOntologyGrounding(srlGrounding))
+    // TODO: make this work - IN PROGRESS
+    //  This attempts to filter out the Theme Process if it is a duplicate of the theme.
+    val tempReturn = Seq(newOntologyGrounding(srlGrounding))
+//    val tuple = srlGrounding.head.predicateTuple
+//    val toReturn = new ArrayBuffer[Seq[OntologyGrounding]]()
+
+    // OntologyGrounding = Map {theme -> List[IndivGr], ...}
+    val bestGrounding: OntologyGrounding = tempReturn.head
```

Comment:
I'm not particularly reviewing this portion, right? bc it's not actually being used (i.e., `tempReturn` is what is returned)... thoughts on that @kwalcock

Reply:
If it is code that would actually run in production, then I'm glad it isn't being executed.  However, if it is standing by waiting to aid in debugging, it will be difficult to maintain.  Sometimes I throw things into a def and just never call it.  The overhead seems acceptably low and one can be sure the code continues to track other changes, like renaming.  (IMO)